### ISSUE TITLE
FindMarkers.default() is not passing down "..."

### ISSUE THREAD

**Issue by TobiTekath** [satijalab/seurat#1978]
Hi,

I just noticed that in `FindMarkers.default()` no extra arguments are passed down to the DE calling tools.  
In the `de.results <- switch(EXPR = test.use, ...)` all the calls are missing the `...` parameter (except for `WilcoxDETest`).

Thanks.

**Comment by andrewwbutler** [comment]
Hi,

The dots are also passed to the Mast and DESeq2 tests. They aren't passed to any of the other tests because there aren't any functions in those tests that would accept other parameters. Is there a specific case where you think they should be passed?

**Comment by TobiTekath** [comment]
Hi, 

thanks for the quick response.
What I am referring to is that additional arguments are actually not passed down to the DE calling functions, although the functions would handle them. 
For example [https://github.com/satijalab/seurat/blob/245d72b5f7c3ae75b185f2ae2d3a5b4c67c61e9a/R/differential_expression.R#L641-L646](https://github.com/satijalab/seurat/blob/245d72b5f7c3ae75b185f2ae2d3a5b4c67c61e9a/R/differential_expression.R#L641-L646)

should probably look like this

```
"DESeq2" = DESeq2DETest(
      data.use = object[features, c(cells.1, cells.2), drop = FALSE],
      cells.1 = cells.1,
      cells.2 = cells.2,
      verbose = verbose,
      ...
    ),
```

**Comment by andrewwbutler** [comment]
Hi,

This was fixed at some point in development between 3.0.2 and 3.1.0. The dots are being passed as you suggest to certain DE test functions. E.g.

https://github.com/satijalab/seurat/blob/c9f2660ee6b88189acc3ff4aa6088a46a2558dd3/R/differential_expression.R#L651-L657

**Comment by TobiTekath** [comment]
Great, thank you very much.
